sources/osgViewer/Viewer.js

```diff
@@ -404,18 +408,12 @@ Viewer.prototype = MACROUTILS.objectInherit( View.prototype, {
         var self = this;
         var render = function () {
             if ( !self.done() ) {
-
-                var isPresentingVR = self._hmd && self._hmd.isPresenting;
-
-                if ( isPresentingVR )
+                if ( self._hmd && self._hmd.isPresenting )
```

Comment:
@stephomi as you are fixing this code ( and sorry for bothering )... would you mind to wrap the requestAnimationFrame in a function in a way that we don't do a comparison every frame? So if you set VR in the viewer self._hmd.requestAnimationFrame is called and by default window.requestAnimationFrame is called. 
I recall to speak about this with @cedricpinson some time ago.